elementary/gtk-3.0/gtk-dark.css

```
@@ -20,7 +20,7 @@
 @import url("brand-dark.css");
 
 /* Default color scheme */
-@define-color base_color #3d4248;
+@define-color base_color @BLACK_500;
```

This feels _quite_ a bit darker which might cause issues. It looks like something in this range is closer to the current brightness of the dark style:

```suggestion
@define-color base_color mix(@BLACK_500, @BLACK_300, 0.5);
```

---

I like that mix() combo. I'll add it to the PR~